### ISSUE TITLE
The "CuyZ\ValinorBundle\Cache\MapperCacheWarmer::warmUp()" method will require a new "string|null $buildDir"

### ISSUE THREAD

**Issue by pchelk1n** [CuyZ/Valinor-Bundle#17]
Hi.
Updated project to Symfony 6.4
Run `bin/console debug:container --deprecations`

```
The "CuyZ\ValinorBundle\Cache\MapperCacheWarmer::warmUp()" method will require a new "string|null $buildDir" argument in the next major version of its interface "Symfony\Component\HttpKernel\CacheWarmer\WarmableInterface", not defining it is deprecated.
      in /app/vendor/symfony/error-handler/DebugClassLoader.php:337
```

**Comment by romm** [comment]
Hi @pchelk1n thanks for the report. I currently don't know how I can solve it without breaking other applications, you can read the discussion [here in the meanwhile](https://github.com/CuyZ/Valinor-Bundle/pull/12#issuecomment-1899046390) — please don't hesitate to help!

**Comment by romm** [comment]
Should be fixed by #18. Thanks for the report @pchelk1n!
